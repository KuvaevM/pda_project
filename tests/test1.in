start=[S]
[S] = `z` | `p`[E];
[E] = `p` | `q` | `l`[E];