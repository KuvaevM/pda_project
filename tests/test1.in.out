States: q0 
Input alphabet: z, p, q, l
Stack alphabet: S, E
Start state: q0
Start stack element: S
Rules: {
	Rule: {
		Left: {
			State: q0
			Input symbol: Terminal(z)
			Stack symbol: NonTerminal(S)
		}
		Right: {
			State: q0
			Stack sequence: Sequence(Empty)
		}
	}
	Rule: {
		Left: {
			State: q0
			Input symbol: Terminal(p)
			Stack symbol: NonTerminal(S)
		}
		Right: {
			State: q0
			Stack sequence: Sequence(NonTerminal(E))
		}
	}
	Rule: {
		Left: {
			State: q0
			Input symbol: Terminal(p)
			Stack symbol: NonTerminal(E)
		}
		Right: {
			State: q0
			Stack sequence: Sequence(Empty)
		}
	}
	Rule: {
		Left: {
			State: q0
			Input symbol: Terminal(q)
			Stack symbol: NonTerminal(E)
		}
		Right: {
			State: q0
			Stack sequence: Sequence(Empty)
		}
	}
	Rule: {
		Left: {
			State: q0
			Input symbol: Terminal(l)
			Stack symbol: NonTerminal(E)
		}
		Right: {
			State: q0
			Stack sequence: Sequence(NonTerminal(E))
		}
	}
}
